.github/workflows/build-wheel.yml

```diff
@@ -0,0 +1,104 @@
+name: build-wheel
+run-name: 'Build Daft wheel for ${{ inputs.os }}-${{ inputs.arch }}-lts=${{ inputs.lts }}'
+
+on:
+  workflow_call:
+    inputs:
+      os:
+        description: The operating system to build for (ubuntu, macos, or windows)
```

Comment:
I think we can make these fields dropdowns or checkboxes so the options are more obvious

Reply:
This action isn't meant to be run manually actually. I'm working on one that will also upload the wheels, which will have dropdowns